README.md

```diff
@@ -22,6 +22,8 @@ The development package allows you to configure the following page components (f
 
 - [CSS](https://github.com/ExLibrisGroup/primo-explore-package/tree/master/VIEW_CODE/css "css documentation")
 
+- [SASS/SCSS](https://github.com/ExLibrisGroup/primo-explore-package/tree/master/VIEW_CODE/scss "sass/scss documentation") (an optional alternative to CSS)
```

Comment:
This link won't work yet since the https://github.com/ExLibrisGroup/primo-explore-package needs to be updated accordingly